lib/persisted-model.js

```diff
@@ -1072,6 +1066,54 @@ PersistedModel.enableChangeTracking = function() {
   }
 };
 
+function rectifyOnSave(ctx, next) {
+  if (ctx.instance) {
+    ctx.Model.rectifyChange(ctx.instance.getId(), reportErrorAndNext);
+  } else {
+    ctx.Model.rectifyAllChanges(reportErrorAndNext);
+  }
+
+  function reportErrorAndNext(err) {
+    if (err) {
+      console.error(
```

Comment:
-1 for hardcoded `console.error()`. I'd prefer just about anything else. At least allow the user to provide a function to call when an error occurs.